site/content/how-to/monitoring/tracing.md

```diff
@@ -0,0 +1,335 @@
+
+- `ObservabilityPolicy`: This resource is a [Policy](https://gateway-api.sigs.k8s.io/reference/policy-attachment/) that targets HTTPRoutes or GRPCRoutes. It is created by the [application developer](https://gateway-api.sigs.k8s.io/concepts/roles-and-personas/) and enables tracing for a specific route or routes. It requires the `NginxProxy` resource to exist in order to complete the tracing configuration.
+
+TODO(sberman): link to reference docs
+
+### Installing NGINX Gateway Fabric with global tracing config
+
+{{< note >}}Ensure that you've already [installed the Gateway API resources]({{< relref "installation/installing-ngf/helm.md#installing-the-gateway-api-resources" >}}).{{< /note >}}
+
+Based on the collector we deployed above, we'll create the following `values.yaml` file to install NGINX Gateway Fabric:
+
+```yaml
+cat <<EOT > values.yaml
+nginx:
+  config:
+    telemetry:
+      exporter:
+        endpoint: otel-collector.tracing.svc:4317
+      spanAttributes:
+      - key: cluster-attribute-key
+        value: cluster-attribute-value
+EOT
+```
+
+We've set the endpoint and added a demo attribute that will be added to all tracing spans.
+
+To install:
+
+```shell
+helm install ngf oci://ghcr.io/nginxinc/charts/nginx-gateway-fabric --create-namespace -n nginx-gateway -f values.yaml
+```
+
+As a result, we should see the following configurations:
+
+```shell
+kubectl get nginxproxies.gateway.nginx.org ngf-proxy-config -o yaml
+```
+
+```yaml
+apiVersion: gateway.nginx.org/v1alpha1
+kind: NginxProxy
+metadata:
+  name: ngf-proxy-config
+spec:
+  telemetry:
+    exporter:
+      endpoint: otel-collector.tracing.svc:4317
+    spanAttributes:
+    - key: cluster-attribute-key
+      value: cluster-attribute-value
+```
+
+```shell
+kubectl get gatewayclasses.gateway.networking.k8s.io nginx -o yaml
+```
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: GatewayClass
+metadata:
+  name: nginx
+spec:
+  controllerName: gateway.nginx.org/nginx-gateway-controller
+  parametersRef:
+    group: gateway.nginx.org
+    kind: NginxProxy
+    name: ngf-proxy-config
+status:
+  conditions:
+  - lastTransitionTime: "2024-05-22T15:18:35Z"
+    message: GatewayClass is accepted
+    observedGeneration: 1
+    reason: Accepted
+    status: "True"
+    type: Accepted
+  - lastTransitionTime: "2024-05-22T15:18:35Z"
+    message: Gateway API CRD versions are supported
+    observedGeneration: 1
+    reason: SupportedVersion
+    status: "True"
+    type: SupportedVersion
+  - lastTransitionTime: "2024-05-22T15:18:35Z"
+    message: parametersRef resource is resolved
+    observedGeneration: 1
+    reason: ResolvedRefs
+    status: "True"
+    type: ResolvedRefs
+```
+
+If you already had NGINX Gateway Fabric installed, then you can simply create the `NginxProxy` resource and link it in the GatewayClass `parametersRef` like shown above, using:
```

Comment:
```suggestion
If you already have NGINX Gateway Fabric installed, then you can create the `NginxProxy` resource and link it to the GatewayClass `parametersRef`:
```

Simplicity is highly subjective, though in general we avoid adjectives altogether.